src/App.jsx

```diff
@@ -108,35 +73,83 @@ function App () {
         window.lodash = _.noConflict()
         setPsiturk(new PsiTurk(turkUniqueId, '/complete'))
         setMethod('mturk')
-        setLoggedIn(true, 'mturk', turkUniqueId)
+        // TODO: This won't work
+        handleLogin('mturk', turkUniqueId)
         /* eslint-enable */
       } else if (config.USE_PROLIFIC) {
         const pID = getProlificId()
         if (config.USE_FIREBASE && pID) {
           setMethod('firebase')
-          setLoggedIn(true, 'prolific', pID)
+          // TODO: This wont work
+          handleLogin('prolific', pID)
```

Comment:
Same question here, is there an issue for this?

Reply:
Added #145